kube/Cargo.toml

```diff
@@ -58,6 +58,7 @@ static_assertions = "1.1.0"
 kube-derive = { path = "../kube-derive", version = "^0.47.0", optional = true }
 jsonpath_lib = "0.2.6"
 tokio-util = { version = "0.6.0", optional = true, features = ["io"] }
+json-patch = "0.2.6"
```

Comment:
this is only for the typesafety of actually parsing the patch as a `json_patch::Patch`, this probably should be an optional dependency behind a `jsonpatch` feature